ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -289,6 +289,18 @@ const triggerSet: TriggerSet<Data> = {
     },
   ],
   timelineReplace: [
+    {
+      'locale': 'en',
+      'replaceSync': {
+		'Astral Glow\/Umbral Glow': 'Astral/Umbral Glow',
+		'Astral Advance\/Umbral Advance': 'Astral/Umbral Advance',
+		'Superchain Coil\/Superchain Burst': 'Superchain Coil/Burst',
+		'Apodialogos\/Peridialogos': 'Apodia/Peridia',
+		'Theos\'s Saltire\/Theos\'s Cross': 'Saltire/Cross',
+		'Astral Impact\/Umbral Impact': 'Astral/Umbral Impact',
+		'Astral Advent\/Umbral Advent': 'Astral/Umbral Advent',
+      },
+    },
     {
```

Comment:
Hopefully the replaceSync is correct here? The single-quotes were escaped using backslashes, as were the forward slashes as I believe the first string is a regex for the find-replace, correct?

Reply:
Yeah, single quotes should be escaped, but no need to escape the forward slash.  Would you mind removing those?

Reply:
I'm assuming it can recognize it's just a string-to-string instead of a regex, somehow? Done and done!

Reply:
My understanding is that forward slash isn't a regex special character, but is only needed when used with `/` notation.  In the sense that, `/foo\/bar/` needs an escape but `new RegExp('foo/bar')` does not (and is the same as `/foo\/bar/`).